layouts/404/single.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
```

Comment:
Can we clean up the format in this file?

Reply:
pushed up a reformatting